### PR TITLE
Update info in kit docs

### PR DESCRIPTION
**Pull request by WillB97**

### CODE REVIEW

File: docs/kit/power-board.md [hunk 1]
```diff
@@ -37,11 +37,11 @@ Finally, there are connectors for external Start and On|Off switches. You may co
 | PWR\|FLAT  | Green when powered<br />Flashing red and green when the battery is low                                            | Green                  |
 | 5V         | Green when 5V is being supplied                                                                                   | Green                  |
 | H0–1, L0–3 | Green when the corresponding output is on [^1]<br />Red when the output's current limit is reached                | Off                    |
-| RUN\|ERROR | Orange on power-up, or USB reset <br />Flashing green when ready to run<br /> Solid green when running or booting | Orange                 |
+| RUN\|ERROR | Orange on power-up, or USB disconnection <br />Flashing green when ready to run<br /> Solid green when running or booting | Orange                 |
 
 [^1]: The outputs only turn on when your program runs (specifically, when the `Robot` object is created).
 
-On power-up, the Power Board will emit some beeps, which are related to the version of the firmware it has installed.
+On power-up and while waiting for the start button, the Power Board will emit a beep.
```
Comment (RealOrangeOne) [inline docs/kit/power-board.md:44]: > while waiting for the start button,

Is this true? This reads to me like board will keep beeping all the while the start button hasn't been pressed

File: docs/kit/servo-board.md [hunk 2]
```diff
@@ -22,7 +22,10 @@ brown wire) at the bottom of the board.
 
 For the servo board to operate correctly, you must connect it to the 12V
 power rail from the power board. A green LED will light next to the
-servo board 12V connector when it is correctly powered.
+servo board 12V connector when it is correctly powered and the 8 outputs 
+on the left-side of the board can be used.
+
+To use the 4 auxillary outputs on the right-side of the board, 5V needs to be connected to the auxillary input.
```
Comment (RealOrangeOne) [inline docs/kit/servo-board.md:28]: > 5V

Question: Is this only true for 5V servos? If 12V servos are used, would 12V not need to be applied?

Comment (WillB97) [reply docs/kit/servo-board.md:28]: We only have 5V servos.

File: docs/kit/power-board.md (hunk 1 above)
Comment (WillB97) [reply docs/kit/power-board.md]: 8389c8d2884a0a054442004249e42f60addd3fa7

**Pull request merged**